packages/astro/src/assets/utils/transformToPath.ts

```diff
@@ -33,3 +34,15 @@ export function hashTransform(
 	);
 	return shorthash(deterministicString(hashFields));
 }
+
+export function getAssetsPrefix(fileType: string, assetsPrefix?: AssetsPrefix): string {
+	if (!assetsPrefix) return '';
+	if (typeof assetsPrefix === 'string') return assetsPrefix;
+	fileType = fileType[0] === '.' ? fileType.slice(1) : fileType;
+	if (assetsPrefix[fileType]) {
+		return assetsPrefix[fileType];
+	} else if (assetsPrefix.fallback) {
+		return assetsPrefix.fallback;
+	}
+	return '';
```

Comment:
I think we can directly return `assetsPrefix.fallback` here because we had validated in the `schema.ts` that it should always exist.